<launch>
  <node name="CsNetworkManager" pkg="network_manager" type="CsNetworkConfigModule.py" />

</launch>
